sdk/servicebus/service-bus/src/core/messageReceiver.ts

```diff
@@ -711,6 +713,57 @@ export class MessageReceiver extends LinkEntity {
     };
   }
 
+  /**
+   * Prevents us from receiving any further messages.
+   */
+  public stopReceivingMessages(): Promise<void> {
+    log.receiver(`[${this.name}] User has requested to stop receiving new messages.`);
```

Comment:
```suggestion
    log.receiver(`[${this.name}] User has requested to stop receiving new messages, attempting to drain the credits.`);
```
or something like that?